roku/core.py

```diff
@@ -57,6 +57,8 @@
 
     # For devices that support being turned on/off
     'power': 'Power',
+    'power_on': 'PowerOn',
+    'power_off': 'PowerOff',
```

Comment:
I don't see PowerOn as a command, but PowerOff will be in, https://github.com/jcarbaugh/python-roku/pull/37, which I plan to test shortly